src/main/java/frc/robot/RobotContainer.java

```diff
@@ -121,19 +120,31 @@ public static double getDriverRightX() {
   private void configureButtonBindings() {
 
     m_driverController.rightStick().whileTrue(new setXCommand());
-    m_driverController.rightTrigger().onTrue(new AmpTrapScore(ManipulatorConstants.PIVOT_AMP_GOAL));
-    m_driverController.rightBumper().onTrue(new ScoreGoal(ManipulatorConstants.PIVOT_FAR_SCORE));
-    m_driverController.leftBumper().onTrue(new ScoreGoal(ManipulatorConstants.PIVOT_CLOSE_SCORE));
+    m_driverController.rightTrigger().onTrue(new Shooter(
+      ManipulatorConstants.SCORE_SIMPLE_PID, 
+      ManipulatorConstants.PIVOT_AMP_GOAL));
+    m_driverController.rightBumper().onTrue(new Shooter(
+      ManipulatorConstants.SCORE_SIMPLE_PID, 
+      ManipulatorConstants.PIVOT_FAR_SCORE));
+    m_driverController.leftBumper().onTrue(new Shooter(
+      ManipulatorConstants.SCORE_SIMPLE_PID, 
+      ManipulatorConstants.PIVOT_CLOSE_SCORE));
     m_driverController.leftTrigger().whileTrue(new Intake(ManipulatorConstants.INTAKE_SPEED));
     m_driverController.a().onTrue(new PivotPID(ManipulatorConstants.PIVOT_MIN));
     m_driverController.x().onTrue(new PivotPID(ManipulatorConstants.PIVOT_FAR_SCORE));
     m_driverController.y().whileTrue(new Intake(ManipulatorConstants.OUTTAKE_SPEED));
     m_driverController.b().whileTrue(new AlignScore());
 
     m_coDriverController.rightStick().whileTrue(new setXCommand());
-    m_coDriverController.rightTrigger().onTrue(new AmpTrapScore(ManipulatorConstants.PIVOT_AMP_GOAL));
-    m_coDriverController.rightBumper().onTrue(new ScoreGoal(ManipulatorConstants.PIVOT_FAR_SCORE));
-    m_coDriverController.leftBumper().onTrue(new ScoreGoal(ManipulatorConstants.PIVOT_CLOSE_SCORE));
+    m_coDriverController.rightTrigger().onTrue(new Shooter(
+      ManipulatorConstants.SCORE_SIMPLE_PID, 
```

Comment:
```suggestion
      ManipulatorConstants.SCORE_SIMPLE_RPM, 
```
this variable should be renamed